core/src/main/java/hudson/model/View.java

```diff
@@ -1000,6 +1005,41 @@ public synchronized void doDoDelete(StaplerRequest req, StaplerResponse rsp) thr
      */
     public abstract Item doCreateItem( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException;
 
+    /**
+     * An API method to get the allowed {$link TopLevelItem}s and its categories.
+     *
+     * @return A {@link Categories} entity that is shown as  JSON file.
+     *
+     * @since TODO
```

Comment:
BTW no need for `@since` on restricted members: no one could use it, thus no one should care when it was added. If and when the restriction is lifted, you add a `@since` for _that_ version.